From cb8b4b9bdd67d4da6e50d1f3257f440b5b3ac7bc Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 13 Oct 2020 20:08:06 +0300
Subject: [PATCH] [SW-0] gaudi: security: remove redundant prints

It is was useful during development, but now useless and just bloats
the log.
Remove printing of each domain RRs' disablement.

Change-Id: I8ea96df25edea0abb90359b960f554373327b6d1
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/security.c b/gaudi/security.c
index a293572..e093bdd 100644
--- a/gaudi/security.c
+++ b/gaudi/security.c
@@ -1189,7 +1189,6 @@
 	regmap_writel(0, mmPCIE_WRAP_RR_RD_PRV_RANGE_REG_MAX_0);
 	regmap_writel(0, mmPCIE_WRAP_RR_RD_PRV_RANGE_REG_MIN_1);
 	regmap_writel(0, mmPCIE_WRAP_RR_RD_PRV_RANGE_REG_MAX_1);
-	preboot_puts("Disabled PCIe RRs\n");
 
 	/* PCIe RTR RR - HBM WP 0x0 - 0x07FFFFFFFF */
 	regmap_writel(0, mmSIF_RTR_CTRL_7_RANGE_PRIV_BASE_LOW_AW_0);
@@ -1208,8 +1207,6 @@
 	/* PCIe RTR RR - Disable (hit) RR */
 	hit = regmap_readl(mmSIF_RTR_CTRL_7_RANGE_PRIV_HIT_AR);
 	regmap_writel(hit & ~BIT(0), mmSIF_RTR_CTRL_7_RANGE_PRIV_HIT_AR);
-
-	preboot_puts("Disabled SIF RRs\n");
 }
 
 void preboot_security_init(void)
